source/postprocess/visualization.cc

```diff
@@ -802,6 +797,14 @@ namespace aspect
                              "and a factor of 8 in 3d, when using quadratic elements for the velocity, "
                              "and correspondingly more for even higher order elements.");
 
+          prm.declare_entry ("Filter output", "false",
+                             Patterns::Bool(),
+                             " deal.II offers the possibility to filter duplicate vertices for hdf5 "
+                             " output files. This merges the vertices of adjacent cells and "
+                             " therefore saves disk space, but misrepresents discontinuous "
+                             " output properties. Activating this function reduces the disk space "
+                             " by about a factor of $2^{dim}$ for hdf5 output.");
```

Comment:
Does this flag have an effect for anything other than HDF5/XDMF output? If not, maybe mention that this.

In any case, I would put a prominent warning into this text:
```
  \\note{\\textbf{Warning:} Setting this flag to true will result in visualization output that does not accurately
  represent discontinuous fields. This may be because you are using a discontinuous finite element for the
  pressure, temperature, or compositional variables, or because you use a visualization postprocessor that
  outputs quantities as discontinuous fields (e.g., the strain rate, viscosity, etc.). These will then all be visualized
  as \\textit{continuous} quantities even though, internally, \\aspect{} considers them as discontinuous fields.}
```